src/caffe/layers/cudnn_conv_layer.cpp

```diff
@@ -51,7 +51,7 @@ void CuDNNConvolutionLayer<Dtype>::LayerSetUp(
       this->num_output_ / this->group_, this->channels_ / this->group_,
       kernel_h, kernel_w);
 
-  this->weight_offset_ = (this->num_output_ / this->group_) *
```

Comment:
We have bunch of other data members initialized in LayerSetup - I just follow them. It's technically not safe but it does require a separate PR to refactor this code.